src/pages/User/components/PullRequests/index.js

```diff
@@ -180,13 +180,16 @@ class PullRequests extends Component {
    */
   displayPullRequests = userInfo => {
     const { data, userDetail } = userInfo;
-    const count = this.countOtherRepos(data, userDetail);
+    const validData = this.getValidPullRequests(data);
+    const otherReposCount = this.countOtherRepos(validData, userDetail);
+
+    this.props.setUserContributionCount(validData.items.length, otherReposCount);
```

Comment:
I doubt this line is still needed? Why? 
I think the ordering of the above lines and the set state below should fix this? Can you check and verify if this is the case?

Reply:
Hmm, sorry i didn't understand , can you tell which lines should be reordered and set state below line ??

Reply:
I mean - I just didn't get why line 186 was needed. All other lines look OK to me. 

Can you remove this line :point_down: and check if you can still make it work? 
```
this.props.setUserContributionCount(validData.items.length, otherReposCount);
```

Reply:
No Dai ! Without setting value in setUserContributionCount, we can't set contribution count for checking the eligibility criteria for showing CompletionMessage or Time message in UsernameForm component . For which on UsernameForm component default state value will be passed 0,0 for totalprCount and otherprCount which will always give display timeMessage.

![Screenshot from 2019-10-26 22-48-38](https://user-images.githubusercontent.com/15843175/67623260-c9e0e580-f842-11e9-828b-168e9e6e2a97.png)